gitpod-io/gitpod
# Update workspace class descriptions

## Description

Update workspace class descriptions per https://github.com/gitpod-io/website/issues/3340.

## Related Issue(s)

Related to https://github.com/gitpod-io/website/issues/3340.

## How to test

1. Add a project in an organization
2. Go to project settings
3. Notice the new description on the workspace class settings

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
